### PR TITLE
Readme improvements

### PR DESCRIPTION
**Pull request by inverse**
- Add badges
- Tidy + typo fix
- Bump year on licence

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -75,4 +79,8 @@ Shell profile configuration file, Shell command alias configuration file, PHP co
 ```
 
 ## Author 
-- [Valay Dave](valaygaurang@gmail.com)
\ No newline at end of file
+- [Valay Dave](valaygaurang@gmail.com)
+
```
Comment (valayDave) [inline Readme.md:83]: @inverse Please Add yourself as contributor too.

File: Readme.md [hunk 2]
```diff
@@ -75,4 +79,8 @@ Shell profile configuration file, Shell command alias configuration file, PHP co
 ```
 
 ## Author
```
Comment (valayDave) [inline Readme.md:81]: Let's change this to Contributors

File: Readme.md (hunk 1 above)
Comment (inverse) [reply Readme.md:83]: Or we could remove this? It's clear from the repo name + new github UI

image


Happy to just leave your name there too!

Comment (valayDave) [reply Readme.md:83]: Yes. Works.

Comment (inverse) [reply Readme.md:83]: Lets keep your name :)

**Pull request merged**